assets/js/set-is-mobile.js

```diff
@@ -0,0 +1,17 @@
+import getCookie from "./cookies/get-cookie";
+import setCookie from "./cookies/set-cookie";
+
+export default function setIsMobile() {
+    let screenWidth = window.innerWidth
+        || document.documentElement.clientWidth
+        || document.body.clientWidth
+        || 0;
+
+    let isMobile = screenWidth < 1024;
```

Comment:
Same could be done with UserAgent detection, but this is better imho.

Reply:
Sorry, but I'm really sure we have to move all this functionality to AffiliateFlask project.
For some reasons - not sure this is the best way to support different view patterns.
Will be glad to see this functionality in a "howto" page format.